uint8 TYPE_REACHING = 0
uint8 TYPE_GRIPPER = 1
uint8 TYPE_PLACEMENT = 2

# Message definition
rr100_ur_worker/Task[] tasks
